Scalar.Installer.Mac/scripts/preinstall

```diff
@@ -7,7 +7,6 @@ function stopService()
         if (/bin/launchctl print $guid/$service | grep --quiet $service); then
                 stop="/bin/launchctl unload /Library/LaunchAgents/$service.plist"
                 echo "Stopping Service: '$service'"
-                eval $stop || exit 1
```

Comment:
did you mean to kill both the eval to actually stop it and the exit-on-error ??

Reply:
Ah. No. I should still try to stop it, but just stop caring about the error. Thanks!

Reply:
seems like we should try to stop it and ignore the error code.